sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -92,14 +92,14 @@ object DottyPlugin extends AutoPlugin {
        */
       def withDottyCompat(scalaVersion: String): ModuleID = {
         val name = moduleID.name
-        if (name != "dotty" && name != "dotty-library" && name != "dotty-compiler")
+        if (name != "scala3" && name != "scala3-library" && name != "scala3-compiler")
           moduleID.crossVersion match {
             case binary: librarymanagement.Binary =>
               val compatVersion =
                 CrossVersion.partialVersion(scalaVersion) match {
                   case Some((3, _)) =>
                     "2.13"
-                  case Some((0, minor)) =>
+                  case Some((0 | 3, minor)) =>
```

Comment:
I don't think that's needed, there's already `case Some((3, _))` above.

Reply:
Oops, you're right 😁